NSTTutorials/UITestCodeGeneration/README.md

```diff
@@ -0,0 +1,74 @@
+[_Return to Tutorials Overview_](../README.md)
+
+# UI Test Code Generation
+
+### Overview
+
+NST can utilize the test methods and request wrappers you have set up in order to export boilerplate code to your UI test files. The assumption behind this functionality is that certain API requests map to specific functionality in a user facing application. For example, you may have a button that when tapped, triggers a list of addresses to load. In this case, we would map the NST service wrapper that returns the response that contains the addresses to tapping the button in the UI.
+
+The benefit provided by this functionality is the ability to run your NST tests and have large portions of your (boilerplate) UI test code generated for you, accelerating the authoring of UI tests.
+
+NST supports **iOS** and **Android** UI test code generation by default, with the understanding (mentioned above) that the platform (iOS / Android) test classes and methods will map 1:1 with the NST test classes and methods. These default output generators ([IosFuiTestLogger](TODO-REPLACE-ME) / [AndroidFuiTestLogger](TODO-REPLACE-ME)) are of course customizable and extendable to suite your specific requirements as well.
+
+### Topics Covered
+
+1. [Runtime arguments](#runtime-arguments)
+2. [Output Mapping CSV](#output-mapping-csv)
+3. [Utilizing the iOS & Android loggers](#utilizing-the-ios--android-loggers)
+4. [Customizing UI test code output](#customizing-ui-test-code-output)
+
+### References
+- [nstToFuiMappingsIos.csv](src/test/resources/nstToFuiMappingsIos.csv)
+- [IosFuiTestLogger](TODO-REPLACE-ME) / [AndroidFuiTestLogger](TODO-REPLACE-ME)
+- [Custom Loggers](../MockGeneration/README.md#adding-a-custom-logger-to-modify-output)
+
+### Prerequisites
+You should have familiarity with the "Adding a custom logger" section in the Mock Generation tutorial. If not, please first review that section [here](../MockGeneration/README.md#adding-a-custom-logger-to-modify-output).
+
+## Runtime arguments
+
+There are several required runtime arguments that need to be set in order to trigger UI test code generation when running your NST tests.
+
+- `whatToWrite` - Should be set to include `TESTS` .
+- `nstplatform` - Should be set to whatever platform you are trying to generate mocks for (e.g. IOS/ANDROID).
+- `androidTestsLocation` / `iosTestsLocation` - Should be set to the directory containing the UI test classes that map to the NST tests.
+
+## Output Mapping CSV
+
+Additionally, we need to create and set up a mapping CSV file with the following columns, with the name of the file prefixed with `nstToFuiMappings`. For example, if creating a mapping file for IOS platform UI test code generation, the file should be called `nstToFuiMappingsIos.csv`. See this module's [resources](src/test/resources) for two example files for iOS and Android.
+
+```csv
+SERVICE,API,PLATFORM_REQUEST_TYPE,NAVIGATION,ENTRY,IMPORTS,MEMBER_FIELDS
```

Comment:
Clearly identify that these are the column headers, in order, for the CSV to adopt.

Reply:
Done in 3d43f0926bdcdf8d1cddbe7d5466662922cf51bb